tests/cli/test_cli_crop.py

```diff
@@ -4,9 +4,15 @@
 import sys
 import tempfile
 
+import hydra
 import torchvision
 import yaml
-from hydra.experimental import compose, initialize
+from hydra.experimental import compose
+
+try:
+    from hydra import initialize
+except ImportError:
+    from hydra.experimental import initialize
```

Comment:
Would be nice to have a comment here that `hydra.initialize` is only available since hydra 1.2 and before it was at `hydra.experimental.intitialize`